### ISSUE TITLE
Can't send sms from calendar

### ISSUE THREAD

**Issue by Hamza-Talibi** [odoo/odoo#20134]
Hello everybody,
When I select a meeting from tree view, I click on the action button in which to open the wizard "Send SMS to attendees", then I get this error :

Error:
Odoo Server Error
```
Traceback (most recent call last):
  File "/home/odoo11/odoo-dev/odoo/odoo/http.py", line 646, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/odoo11/odoo-dev/odoo/odoo/http.py", line 307, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/home/odoo11/odoo-dev/odoo/odoo/tools/pycompat.py", line 86, in reraise
    raise value
  File "/home/odoo11/odoo-dev/odoo/odoo/http.py", line 683, in dispatch
    result = self._call_function(**self.params)
  File "/home/odoo11/odoo-dev/odoo/odoo/http.py", line 339, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/odoo11/odoo-dev/odoo/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/odoo11/odoo-dev/odoo/odoo/http.py", line 332, in checked_call
    result = self.endpoint(*a, **kw)
  File "/home/odoo11/odoo-dev/odoo/odoo/http.py", line 927, in __call__
    return self.method(*args, **kw)
  File "/home/odoo11/odoo-dev/odoo/odoo/http.py", line 512, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo11/odoo-dev/odoo/addons/web/controllers/main.py", line 916, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/home/odoo11/odoo-dev/odoo/addons/web/controllers/main.py", line 908, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/home/odoo11/odoo-dev/odoo/odoo/api.py", line 687, in call_kw
    return call_kw_model(method, model, args, kwargs)
  File "/home/odoo11/odoo-dev/odoo/odoo/api.py", line 672, in call_kw_model
    result = method(recs, *args, **kwargs)
  **File "/home/odoo11/odoo-dev/odoo/addons/sms/wizard/send_sms.py", line 41, in default_get
    number = partner.mobile
AttributeError: 'calendar.event' object has no attribute 'mobile'**
```
Any help please!

Thank you

**Comment by milanpatel1519** [comment]
Also '_' import missing.

Link: https://github.com/odoo/odoo/blob/11.0/addons/sms/wizard/send_sms.py#L48

**Comment by Hamza-Talibi** [comment]
Hi @Yenthe666 have you seen this error ?

**Comment by Yenthe666** [comment]
@Hamza-Talibi - I have but haven't had time to reproduce (and no test account).
I did handle the second bug regarding translations already.

**Comment by mart-e** [comment]
Thanks for the report, fixed at d0e165004246b77608a4c7017594332e5c8ab45f
